src/common/common_utilities.py

```diff
@@ -7199,14 +7245,15 @@ def getAviIpFqdnDnsMapping(avi_controller_fqdn_ip_dict, dns_server):
     try:
         for dns in dns_server:
             for avi_fqdn, avi_ip in avi_controller_fqdn_ip_dict.items():
-                listOfCmd = ["dig", dns, avi_fqdn, "+short"]
+                listOfCmd = ["dig", f"@{dns}", avi_fqdn, "+short"]
```

Comment:
Since ` dnspython` is being used elsewhere, what are your thoughts on shoehorning in `Resolver` here? Totally can see it as a being out of scope of this PR (this looks to be purely for :bug: squashing), but it's so much more efficient than shelling out to run `dig` and parsing the output.